back/src/bsvhu/elastic.ts

```diff
@@ -15,13 +15,13 @@ type WhereKeys =
 // | state              | emitter | transporter | destination |
 // |--------------------|---------|-------------|-------------|
 // | initial (draft)    | draft   | draft       | draft       |
-// | initial            | action  | to collect  | follow      |
+// | initial            | action  | follow      | follow      |
```

Comment:
Je me permets juste d'attirer votre attention sur le fait qu'il semblerait que c'était volontaire d'afficher le vhu dans la section "to collect". Je vois pas pourquoi mais si quelqu'un sait qu'il se manifeste vigoureusement